src/main/java/org/opentripplanner/routing/graph/Edge.java

```diff
@@ -28,10 +32,29 @@
      */
     private static final UniqueIdGenerator<Edge> idGenerator = new IncrementingIdGenerator<Edge>();
 
+    public long getClusterId() {
+        return clusterId;
+    }
+
+    public void setClusterId(long clusterId) {
+        this.clusterId = clusterId;
+    }
+
     /**
      * Identifier of the edge. Negative means not set.
      */
     private int id;
+    private long  clusterId;
+
+    public ArrayList<TimeTable> getTimes() {
+        return times;
+    }
+
+    public void setTimes(ArrayList<TimeTable> times) {
+        this.times = times;
+    }
+
+    ArrayList<TimeTable> times;
```

Comment:
Wszystkie te zmiany powinny trafić do `StreetEdge`, a nie do `Edge`. Jest mnóstwo krawędzi, które nie są ulicami, także ten kod dostępny w nich jest mylący

Reply:
ok zronie to jak tylko uzyslam pewnosc ze wszystko dziala